src/util/pg.py

```diff
@@ -212,10 +212,11 @@ def explode_query_range(cr, query, table, alias=None, bucket_size=10000, prefix=
         parallel_filter = "{alias}.id IS NOT NULL".format(alias=alias)
         return [query.format(parallel_filter=parallel_filter)]
 
-    parallel_filter = "{alias}.id BETWEEN %s AND %s".format(alias=alias)
+    parallel_filter = "{alias}.id BETWEEN %(low)s AND %(high)s".format(alias=alias)
```

Comment:
Nitpicking on variable name:
```suggestion
    parallel_filter = "{alias}.id BETWEEN %(lower-bound)s AND %(upper-bound)s".format(alias=alias)
```

Reply:
Done